src/test/resources/processExecution/recorder.ts

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as child_process from "child_process";
+import * as events from "events";
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+
+interface ISpawnOptions {
```

Comment:
I think the NodeJS typings for child_process already have the full interface definition for these options, so you shouldn't have to define your own